packages/web-components/src/appointment-planner/client/core/theme.ts

```diff
@@ -0,0 +1,13 @@
+/* istanbul ignore file */
+import { injectGlobal } from 'emotion'
+
+export interface InitializerTheme {
+  hoverBackgroundColor?: string
+}
+
+export const injectGlobalCss = ({ hoverBackgroundColor }: InitializerTheme) => injectGlobal`
```

Comment:
No globals please, the theme is handled by adding custom classes scoped to the container id